sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/UnitTestResultsImportSensor.java

```diff
@@ -98,14 +98,6 @@ void analyze(SensorContext context, UnitTestResults unitTestResults) {
         .withValue(executionTime)
         .save();
     }
-
-    if (aggregatedResults.tests() > 0) {
```

Comment:
There is a warning in logs for quite a long time saying it is deprecated + this value is ignored by the server.